aplib.net-demo/Assets/Scripts/Tiles/Corner.cs

```diff
@@ -5,23 +5,25 @@ namespace Assets.Scripts.Tiles
     /// <summary>
     /// Represents a corner tile.
     /// ___ ___
-    /// | | |_|
-    /// | |____
+    /// |_| | |
+    /// ____| |
     /// |_____|
     /// </summary>
     public class Corner : Tile
     {
         /// <summary>
         /// Initializes a new instance of the <see cref="Corner"/> class.
-        /// The default is a top-right corner.
+        /// The default is a top-left corner.
         /// </summary>
         /// <param name="rotate">The amount of times to rotate the tile.</param>
         public Corner(int rotate = 0)
         {
+            rotate = (rotate + 4) % 4; // Normalise to [0..3]
```

Comment:
Why are we adding 4. For modularity that doesn't add up.

Reply:
Is this to deal with negative numbers? If so, we could make a helper method for this that checks if `i % m` is negative and just adds `m`.

Reply:
It is for negatives! And I knew it could be done more elegantly, thankd

Reply:
This is a mistake on my part because I did not thought about this but maybe it is better to use an Enum with North, East, South, West. This way we don't need to check for negative values.

Reply:
Since 995efebd5f2d2c45c477809db015ff0a0bdc71d1, we now use directions everywhere. No more numbers!